code/datums/ai/oldhostile/hostile_tameable.dm

```diff
@@ -109,7 +106,7 @@
 
 	if(!COOLDOWN_FINISHED(src, command_cooldown))
 		return
-	if(!istype(clicker) || blackboard[BB_HOSTILE_FRIEND] == WEAKREF(clicker))
+	if(!istype(clicker) || blackboard[BB_HOSTILE_FRIEND] != clicker)
```

Comment:
was this broken? you appear to have changed the intention of this check from "equals" to "does not equal". This switch isn't seen in places like line 142 of this same file.

Reply:
alright that's cool just making sure this wasn't some type of fat-finger